R/assertions.R

```diff
@@ -1157,7 +1206,11 @@ assert_unit <- function(dataset,
       class = c(class, "assert-admiraldev")
     )
   }
-  if (tolower(units) != tolower(required_unit)) {
+
+  if (!is.null(required_unit) && !(tolower(units) %in% tolower(required_unit))) {
+    # change cli `.val` to end with OR instead of AND
+    divid <- cli_div(theme = list(.val = list("vec-last" = ", or ", "vec_sep2" = " or ")))
+
     message <-
       message %||%
       "It is expected that {.val {param}} has unit of {.val {required_unit}}.
```

Comment:
```suggestion
      "It is expected that {.val {param}} has unit of {.or {required_unit}}.
```

Reply:
Unfortunately, this is not equivalent:
```
> values <- c("a", "b")
> cli::cli_inform(c(".or: {.or {values}}", ".val: {.val {values}}"))
.or: a or b
.val: "a" and "b"
```
Thus I would keep `.val`.

Reply:
Gotcha! Fine then.

Reply:
Ok, makes sense